config/config-default.json

```diff
@@ -10,6 +10,7 @@
     "files-scs:config/presets/ldp/request-parser.json",
     "files-scs:config/presets/setup.json",
     "files-scs:config/presets/storage.json",
-    "files-scs:config/presets/storage_wrapper.json"
+    "files-scs:config/presets/storage_wrapper.json",
+    "files-scs:config/presets/variables.json"
```

Comment:
Maybe give a more descriptive name? Here it can be `runtime-settings` or `cli-params` for me.